weapons.yaml

```diff
@@ -597,6 +597,20 @@ MammothTusk:
 		ImpactSound: gexpwasa.wav
 		ValidImpactTypes: Water
 
+Maverick:
+	ReloadDelay: 10
+	Range: 6c0
+	Report: vintatta.wav
+	ValidTargets: Ground
+	Burst: 2
```

Comment:
I am not sure how to setup the weapons correctly so I left that open for now. Guess it can't work due to voxel projectiles not being implemented. Removed the burst.